imagelab/DynaPanel.java

```diff
@@ -20,96 +20,105 @@
     /** Default height of the title text. */
     public static final int TITLE_HEIGHT = 40;
     /** Default background color. */
-    public static final Color FRAME_BG_COLOR = new Color(255,0,0,0);
-    
+    public static final Color FRAME_BG_COLOR = new Color(255, 0, 0, 0);
+
     /** Singleton object. */
     private static DynaPanel dynPanSingleton;
 
     /**
      * Factory method to get the canvas singleton object.
+     *
+     * @return the DynPanSingleton Object
      */
     public static DynaPanel getDynPan() {
-        if(dynPanSingleton == null) {
-            dynPanSingleton = new DynaPanel((Image)null);
+        if (dynPanSingleton == null) {
+            dynPanSingleton = new DynaPanel((Image) null);
         }
         dynPanSingleton.setVisible(true);
         return dynPanSingleton;
     }
 
-    // Instance Variables
+    /**Instance Variable for background color. */
     private Color backgroundColour;
+    /**Instance Variable for the panel image. */
     private Image panelImage;
+    /**Instance Variable for the graphic. */
```

Comment:
Missing space after open comment marker.  (Also, no need to state that this is an "Instance Variable".)